instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_client.py

```diff
@@ -162,7 +146,8 @@ def _intercept_server_stream(
         else:
             mutable_metadata = OrderedDict(metadata)
 
-        with self._start_span(client_info.full_method) as span:
+        span = self._start_span(client_info.full_method)
+        with trace.use_span(span, end_on_exit=True):
```

Comment:
This is equivalent to the previous code I think

Reply:
I've reverted `_start_span` to call `start_as_current_span` rather than `start_span`